doc/cephfs/multifs.rst

```diff
@@ -29,6 +29,19 @@ support the new file system. The deployment technology used, e.g. cephadm, will
 also configure the MDS affinity (see: :ref:`mds-join-fs`) of new MDS daemons to
 operate the new file system.
 
+In case the data and/or metadata pools for the volume/CephFS that needs to be
+created are already present, name(s) of these pools can be passed as follows::
+
+    ceph fs volume create <vol-name> --meta-pool <meta-pool-name> --data-pool <data-pool-name>
+
+And if a volume has to have multiple data pool names that can be done by
+passing all the name separated by comma::
+
```

Comment:
Multiple data pools can be specified with a comma-separated list:

Reply:
Fixed.